doc/_admin-guide/README.md

```diff
@@ -2,7 +2,7 @@
 title: syslog-ng Open Source Edition Administration Guides
 id: adm-guide
 description: >-
-    Welcome to the {{ site.product.name }} {{ site.github.latest_release.name | remove: '.tar.gz' | remove: 'syslog-ng-'}} Administration Guide. This document describes how to configure and manage {{ site.product.name }} ({{ site.product.short_name }}). Background information for the technology and concepts used by the product is also discussed. 
+    Welcome to the {{ site.product.name }} 4.8.1 Administration Guide. This document describes how to configure and manage {{ site.product.name }} ({{ site.product.short_name }}). Background information for the technology and concepts used by the product is also discussed. 
```

Comment:
Could you please add a (placeholder) function instead that just echoes this constant string 4.8.1 (and will enhance it later to do the trick properly)